### PR TITLE
Loading class

### PR DESCRIPTION
**Pull request by vandangnhathung**
Build up 2 options for loading class
- `enable: boolean` (True => loading is used in the project)
- `isWoocommerce: boolean`  (True => Woocommerce is used in the project)

### CODE REVIEW

File: scss/_config.scss [hunk 1]
```diff
@@ -40,7 +40,13 @@
...
 // vertical spacing
-//$vertical_spacing: (
-//        "values":0 1 2 4 6 8 12 16 24 32 48 64 96 128,
-//        "md":    0 1 2 4 6 8 12 16 24 30 40 50 60 70,
-//);
\ No newline at end of file
+$vertical_spacing: (
+        "values":0 1 2 4 6 8 12 16 24 32 48 64 96 128,
+        "md": 0 1 2 4 6 8 12 16 24 30 40 50 60 70,
+);
```
Comment (phucbm) [inline scss/_config.scss:46]: Changes of vertical spacing is redundant in this PR.

File: css/loading-class.css
Comment (phucbm) [inline css/loading-class.css:1]: Run production build to generate this file.

File: scss/_config.scss [hunk 2]
```diff
@@ -40,7 +40,13 @@
...
+
+// loading class
+$loading_class: (
+        "enable": false,
+        "isWoocomerce": true // true if project using woocommerce
```
Comment (phucbm) [inline scss/_config.scss:51]: The default value of `enable` should be `true`. The default value of `isWoocommerce` should be `false`.

File: scss/loading-class.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+ * Loading
+ */
+@import "helpers";
+
+@mixin print-loading-class($data) {
+  $isEnabled: map-get($data, 'enable');
+  $isWoocommerce: map-get($data, 'isWoocomerce');
+
+  // $isEnabled: true => loading is used
+  @if ($isEnabled) {
```
Comment (phucbm) [inline scss/loading-class.scss:11]: Avoid nested `if` as much as possible.

File: scss/loading-class.scss [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    // If $isWoocommerce value is false just load the loading selector only
+    @if ($isWoocommerce == false) {
```
Comment (phucbm) [inline scss/loading-class.scss:13]: Avoid negative comparisons when possible.

**Review by phucbm — CHANGES_REQUESTED** [review]
Please check my comments.

File: scss/_config.scss [hunk 5]
```diff
@@ -40,13 +40,13 @@
...
 
 // vertical spacing
-$vertical_spacing: (
-        "values":0 1 2 4 6 8 12 16 24 32 48 64 96 128,
-        "md": 0 1 2 4 6 8 12 16 24 30 40 50 60 70,
-);
+//$vertical_spacing: (
+//        "values":0 1 2 4 6 8 12 16 24 32 48 64 96 128,
+//        "md": 0 1 2 4 6 8 12 16 24 30 40 50 60 70,
+//);
```
Comment (phucbm) [inline scss/_config.scss:46]: Unrelated changes.

**Review by phucbm — CHANGES_REQUESTED** [review]

File: scss/_config.scss (hunk 5 above)
Comment (vandangnhathung) [reply scss/_config.scss:46]: image
I don't know whether this is the change of code in the image above. 

It doesn't show any changes when I compare my local with `staging` branch in the image below
image

**Review by phucbm — APPROVED** [review]

**Comment by phucbm** [comment]
Thank you @vandangnhathung
